src/main/webapp/index.html

```diff
@@ -48,13 +48,23 @@ <h1>About our Team</h1>
         Drew is a third-year student studying computer science at UT Austin.
         He enjoys working out, basketball, traveling, and recreational ping-pong.
         <br /> &emsp;&emsp;
+<<<<<<< HEAD
```

Comment:
It looks like you have an unresolved merge conflict here. Remove everything from here to the ======= and then remove the '>>>>>>> 4ba7decc2f1b5ed85204952c3e065b93f73ea35e' line down below.